features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/blocks/application/application-add/ajax/application-add.jag

```diff
@@ -53,6 +53,12 @@ include("/jagg/jagg.jag");
             callbackUrl = request.getParameter("callbackUrl");
         }        
         description = request.getParameter("description");
+        if (request.getParameter("applicationAttributes") == null) {
```

Comment:
IMO, we do not have to specifically check == null, rather we can use it as following,
if (request.getParameter("applicationAttributes")) {

Reply:
Yes, it can be used as if (request.getParameter("applicationAttributes")) {.
Fixed in #6098